internal/eval/commands.go

```diff
@@ -620,6 +620,13 @@ var (
 		Arity:    -4,
 		KeySpecs: KeySpecs{BeginIndex: 1},
 	}
+	hkeysCmdMeta = DiceCmdMeta{
+		Name: "HKEYS",
+		Info:  `HKEYS command is used to retrieve all the keys(or field names) within a hash. Complexity is O(n) where n is the size of the hash.`,
+		Eval: evalHKEYS,
+		Arity: -1,
```

Comment:
Since HKEYS only accepts key as the argument, we can keep Arity as 1.

Reply:
it was a typo will fix it.